error.go

```diff
@@ -13,6 +13,9 @@ const (
 	E_UNSUPPORTED_GRANT_TYPE           = "unsupported_grant_type"
 	E_INVALID_GRANT                    = "invalid_grant"
 	E_INVALID_CLIENT                   = "invalid_client"
+
+	// Accelbyte OAuth Extensions Errors
+	E_RATE_LIMIT_EXCEEDED              = "rate_limit_exceeded"
```

Comment:
I think we do  not need treat it as Accelbyte's special error. This is a common OAuth error.